.github/workflows/ansible.yml

```diff
@@ -134,7 +134,6 @@ jobs:
           - ansible~=6.0
           - ansible~=7.0
         distro:
-          - "2012"
```

Comment:
Update [readme](https://github.com/signalfx/splunk-otel-collector/tree/main/deployments/ansible#windows) to reflect this?

Reply:
done